src/main/java/school/hei/haapi/model/Event.java

```diff
@@ -0,0 +1,68 @@
+package school.hei.haapi.model;
+
+import static jakarta.persistence.EnumType.STRING;
+import static jakarta.persistence.GenerationType.IDENTITY;
+import static org.hibernate.type.SqlTypes.NAMED_ENUM;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.JoinTable;
+import jakarta.persistence.ManyToMany;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.time.Instant;
+import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import org.hibernate.annotations.JdbcTypeCode;
+import school.hei.haapi.endpoint.rest.model.EventType;
+
+@Entity
+@Table(name = "\"event\"")
+@Getter
+@Setter
+@ToString
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class Event {
+
+  @Id
+  @GeneratedValue(strategy = IDENTITY)
+  private String id;
+
+  @JdbcTypeCode(NAMED_ENUM)
+  @Enumerated(STRING)
+  private EventType type;
+
+  private String description;
+
+  @Column(name = "begin_datetime")
+  private Instant begin;
+
+  @Column(name = "end_datetime")
+  private Instant end;
```

Comment:
use endDatetime as variable name, same for begin -> beginDatetime etc ( for api also)